### ISSUE TITLE
index window is not open in IOS

### ISSUE THREAD

**Issue by swijaya** [yomybaby/kr.yostudio.drawer#15]
Hi,

the index window didnt open in IOS  if i place the menu view under the next level of the widget. example like the view below but it worked in android. 
<Alloy>
    <Widget src="kr.yostudio.drawer"   onDraweropen="onDrawerOpen"    onDrawerclose="onDrawerClose">

```
    <View class="menuWrap" role="leftView">
        <Require id="menuC" src="menu">
        </Require>
    </View>         
    <Window platform="ios" role="centerWindow" title="yo.drawer widget">    
        <LeftNavButton>
            <View>
                <Button backgroundColor="black" onClick="onMenuButtonClick"/>
            </View>
        </LeftNavButton>
        <Require src="main" id="mainC">
        </Require>
    </Window>

</Widget>
```

</Alloy>

But the index able to open if i move the menu view into window view and then i will get these error below when i click on the menu button 

[ERROR] :      message = "undefined is not an object (evaluating 'this._view.scrollView.views[this._view.scrollView.currentPage ? 0 : 1]')";
[ERROR] :      stack = "toggleLeftWindow\ntoggleLeftView\nonMenuButtonClick";

The Titanium SDK i used is 5.4.0GA.

**Comment by nuno** [comment]
Hello @swijaya  check if it is related https://github.com/yomybaby/kr.yostudio.drawer/issues/16 ?

**Comment by yomybaby** [comment]
I just solved #16 , @swijaya Can you try it again with new version?
